{"uuid": "9b9c535d-6f73-46eb-a57d-640a77dd6761", "children": ["8d7354a6-b2b4-493b-bfcb-0a16895e45fc"], "befores": [{"name": "login_to_affairs", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.InvalidSessionIdException: Message: invalid session id: session deleted as the browser has closed the connection\nfrom disconnected: not connected to DevTools\n  (Session info: chrome=137.0.7151.69)\nStacktrace:\n\tGetHandleVerifier [0x0x7ff6e4aa6f65+78965]\n\tGetHandleVerifier [0x0x7ff6e4aa6fc0+79056]\n\t(No symbol) [0x0x7ff6e4839dda]\n\t(No symbol) [0x0x7ff6e4825bc5]\n\t(No symbol) [0x0x7ff6e484ac04]\n\t(No symbol) [0x0x7ff6e48c0195]\n\t(No symbol) [0x0x7ff6e48e06cd]\n\t(No symbol) [0x0x7ff6e48b8443]\n\t(No symbol) [0x0x7ff6e4881311]\n\t(No symbol) [0x0x7ff6e48820a3]\n\tGetHandleVerifier [0x0x7ff6e4d5e26d+2926461]\n\tGetHandleVerifier [0x0x7ff6e4d58993+2903715]\n\tGetHandleVerifier [0x0x7ff6e4d76aed+3026941]\n\tGetHandleVerifier [0x0x7ff6e4ac16fe+187406]\n\tGetHandleVerifier [0x0x7ff6e4ac96ef+220159]\n\tGetHandleVerifier [0x0x7ff6e4aafaf4+114692]\n\tGetHandleVerifier [0x0x7ff6e4aafca9+115129]\n\tGetHandleVerifier [0x0x7ff6e4a964d8+10728]\n\tBaseThreadInitThunk [0x0x7ffee1537374+20]\n\tRtlUserThreadStart [0x0x7ffee1a7cc91+33]\n\n", "trace": "  File \"D:\\Program Files\\Python310\\lib\\site-packages\\pluggy\\_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n  File \"D:\\Program Files\\Python310\\lib\\site-packages\\pluggy\\_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"D:\\Program Files\\Python310\\lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"D:\\Program Files\\Python310\\lib\\site-packages\\pluggy\\_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"D:\\Program Files\\Python310\\lib\\site-packages\\_pytest\\fixtures.py\", line 1195, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"D:\\Program Files\\Python310\\lib\\site-packages\\_pytest\\fixtures.py\", line 922, in call_fixture_func\n    fixture_result = next(generator)\n  File \"D:\\PycharmProjects\\APS\\Tests\\test_system\\test_affairs.py\", line 49, in login_to_affairs\n    page.login(date_driver.username, date_driver.password, date_driver.planning)\n  File \"D:\\PycharmProjects\\APS\\Pages\\itemsPage\\login_page.py\", line 36, in login\n    self.click_login_button()\n  File \"D:\\PycharmProjects\\APS\\Pages\\itemsPage\\login_page.py\", line 39, in click_login_button\n    self.click_button('//button[contains(@class, \"ivu-btn-primary\")]')\n  File \"D:\\PycharmProjects\\APS\\Pages\\itemsPage\\login_page.py\", line 24, in click_button\n    self.click(By.XPATH, xpath)\n  File \"D:\\PycharmProjects\\APS\\Pages\\base_page.py\", line 45, in click\n    element = WebDriverWait(self.driver, wait_time).until(\n  File \"D:\\Program Files\\Python310\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 137, in until\n    value = method(self._driver)\n  File \"D:\\Program Files\\Python310\\lib\\site-packages\\selenium\\webdriver\\support\\expected_conditions.py\", line 633, in _predicate\n    target = driver.find_element(*target)  # grab element at locator\n  File \"D:\\Program Files\\Python310\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 898, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n  File \"D:\\Program Files\\Python310\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 429, in execute\n    self.error_handler.check_response(response)\n  File \"D:\\Program Files\\Python310\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1761119578923, "stop": 1761119588074}], "start": 1761119578923, "stop": 1761119588437}